static/js/store/configureStore.ts

```diff
@@ -20,7 +19,7 @@ export default function configureStore(initialState?: ReduxState) {
   let createStoreWithMiddleware
   if (process.env.NODE_ENV !== "production" && !global._testing) {
     createStoreWithMiddleware = compose(
-      applyMiddleware(...COMMON_MIDDLEWARE, createLogger()),
```

Comment:
I've always found our redux logging annoying and not helpful...it just clutters the dev console. In general, the same information is available in redux dev tools ([chrome](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en); [firefox](https://addons.mozilla.org/en-US/firefox/addon/reduxdevtools/)

<img width="1048" alt="Screenshot 2022-12-23 at 11 02 52 AM" src="https://user-images.githubusercontent.com/9010790/209364669-6c08dabf-5228-4a91-a0bc-20115b808a56.png">

It was sufficiently annoying while debugging this issue that I just got rid of it, and I think that should be a permanent change.